RInAppMessaging/Classes/RInAppMessaging.swift

```diff
@@ -1,3 +1,5 @@
+import Foundation
```

Comment:
Is that import required?

Reply:
Sometimes when I clean the project I get a warning/error for a moment that I'm using `Foundation` features (`@objc`) without importing it. It works without that import in our app but it might not for others.